Sips 2025 version

DaniMori opened the pull request:
Create SIPS 2025 version of the slide show and additional materials

JesusSierralaya committed:
Adds co-speaker names

JesusSierralaya committed:
Updates help slide to reflect in-person support

DaniMori committed:
Reorder "adding new files" and "editing README.md"

Adding and editing scripts is more significant for learners and will better represent their typical git workflow.
This will hopefully help them better learn the usual git workflow.

DaniMori committed:
Update description and software versions

Makes description better reflect the content (and versions) of the repository.
Also, by knitting "README.Rmd", all the software component versions (R, Rstudio, and git) are automatically updated.

DaniMori committed:
Update "viewing deleted files"

No files have been deleted up to now, so the slides didn't make sense as they were.

DaniMori committed:
Put "committing" section after "viewing changes"

It makes more sense to keep on working on the "git basics" before showing how to add a dataset and make further changes.

DaniMori committed:
Move "Using dataset" to further practice session

First, it seems better to teach the whole git workflow

DaniMori committed:
Correct "loop indexing" in slide

After moving around the "Using a dataset" section the "Making your repo reusable" slide looped to the beginning, (possibly) due to some (unknown) indexing error.

Also, the size text is made "smaller" to fit in the slide window.

DaniMori committed:
Correct style in "making code reusable" slide

Improve the slide style to reflect clearly the steps with bullet points, and fit better in the slide

DaniMori committed:
Update "viewing online" to new ordering

After reordering the materials it didn't make sense to see first the "repo main page". In the "syncing with GitHub" slide only the changes to the file added are shown, and the part about updating the README.md is moved to the "making repo reusable" slide

DaniMori committed:
Merge pull request #25 from DaniMori/improvement/reorder-practice

Reorder "adding new files" and "editing README.md"

DaniMori committed:
Update facilitators info

DaniMori committed:
Update title slide details (event logo and date)

DaniMori committed:
Update "Participate" info

New facilitators and use of stickers for "in-person" version

DaniMori committed:
Merge branch 'draft/sips-2025-version' into presentation-minor-changes

Commit:
Update "git configuration" slide

Uses {git2r} in R console instead of terminal, and adds examples of how to fill in the values for the email and user name.

Commit:
Add @Jesussierralaya's ORCID

Commit:
Restore sample script to mortality analysis

We need an example dataset and script that is relatively meaningful for psychology academics.
Reverting the changes in the dataset and script for the SORTEE version will allow using the "mortality example", expected to be more aligned in meaning with the attendees interests.

Commit:
Update script example information in README file

Commit:
Change slides to point to new dataset

NOTE: bit.ly link will not work until this branch is merged to "main"!

Commit:
Re-writes analysis script as notebook

Commit:
Add changes in slides to refer to notebook

Commit:
Drop issues that do not apply to SIPS version

Drop "pasting from Zoom" (only applies to online version) and {pak} problem (it is deactivated to avoid the issue)

DaniMori committed:
Merge pull request #34 from DaniMori/documentation/drop-issues

Drop issues that do not apply to SIPS version

DaniMori committed:
Merge pull request #33 from DaniMori/improvement/notebook

Re-writes analysis script as notebook

DaniMori committed:
Merge pull request #31 from DaniMori/improvement/git-configuration

Update "git configuration" slide

DaniMori committed:
Merge pull request #26 from DaniMori/improvement/repo-description

Update description and software versions

Commit:
Center facilitator names

Commit:
Add @JesusSierralaya's institutional logo

And also adjust the other logos

Commit:
Drops redundant "get help" slide

- Answers https://github.com/DaniMori/git-workshop/pull/12#discussion_r2115914763

Commit:
Make "participate" section fratment

DaniMori committed:
Merge pull request #12 from DaniMori/presentation-minor-changes

Change co-facilitator support info, and other changes related to the workshop logistics

Commit:
Update self-download link

Commit:
Update notes to address different browser issues

Commit:
Add note section headers

Add section headers in notes, to help the instructor(s) have a clearer idea of what part of a slide each note refers to

DaniMori committed:
Merge pull request #35 from DaniMori/improvement/osf-link

Update self-download link

DaniMori committed:
Merge branch 'draft/sips-2025-version' into improvement/note-sections

DaniMori committed:
Merge pull request #36 from DaniMori/improvement/note-sections

Add note section headers

Commit:
Drop note about "adding collaborators" later on

This is a remain from a previous, longer version of the workshop

Commit:
Add note about using personal name / description

For own projects it's better to chose their own

Commit:
Explaining collaboration in private repos

Includes changing the style (to "smaller") to fit in the new content

Commit:
Correct "renv" explanation slide

It overflowed after chaning the default height for code blocks

Commit:
Add image to "make changes" section title slide

Cristrinaranjus reviewed (changes requested):
Some minor changes to go :-)

Commit:
Correct README file & update file system structure

Makes some minor fixes to the README.md file, including fixes to:

- https://github.com/DaniMori/git-workshop/pull/37#discussion_r2123295371

- https://github.com/DaniMori/git-workshop/pull/37#discussion_r2123317080

Also, it updates the folder structure (adding "notebooks" and deleting "src" folder, adds "OSF" as a storage service (in the knitting instructions), and updates the line to update the setup slide with the link to the hosted "audience version".

DaniMori committed:
Make "commented line" work straight away

Co-authored-by: Cris Rodríguez <48205582+Cristrinaranjus@users.noreply.github.com>

Commit:
Add comment about extra packages for {gtsummary}

It may be odd to attendees to install these two packages when they are not used, so an explanation is provided.

JesusSierralaya reviewed (dismissed):
I agree with all changes. Thank you for your work!

Commit:
Add @Cristinaranjus' institutional logo

- Adresses https://github.com/DaniMori/git-workshop/pull/37#discussion_r2123344609

Commit:
Update UAM logo

Makes the visual weight more balanced among the institutional logos.

Commit:
Add "Analysis.R" dependencies

Commit:
Apply sentence wrapping to notebook

Sentence wrapping is easier to understand and use in VC, so better provide it in that format to workshop attendees.

(Also, note this changes the line numbers, so that must be updated in line 880 of "output/slide-deck.qmd".)

Commit:
Update title to the one for SIPS-2025 edition

Commit:
Add note about start Rstudio choosing R version

Commit:
Add indications for presenter changes

Commit:
Explains use of local folders for cloning better

Commit:
Explains repository folder structure before ".git"

Commit:
Add headers to file explanations in speaker notes

Commit:
Wrap "warning" about "discard chunk" in callout

Commit:
Add notes about staging by line/chunk

Commit:
Correct errata in configuration of user email

It used argument `user.name` instead of `user.email`.

Commit:
Move earlier the "Viewing history" slide

Commit:
Correct note about viewing file changes in history

DaniMori committed:
Merge pull request #40 from DaniMori/improvement/post-test-update

Add post-test corrections

Commit:
Correct "notes" div

The div was labelled as "note", making the "notes" text appear in the slide

DaniMori committed:
Clarify use of "saging" files, chunks & lines

Better not to explain it in the first slide (it may be confusing)

DaniMori committed:
Add further explanations in notes

Explains important motivation for certain steps that need to be done

DaniMori committed:
Moves "Viewing history" to the "Practice" session

Viewing history this early makes no sense, as there are no changes to files that can be navigated to and/or restored.

DaniMori committed:
Add clarification about the meaning of a "commit"

To make sure people don't mistake "adding things" to "adding changes to a commit". Otherwise, it is difficult to understand that "adding a deleted file" is also "adding a change to a commit".

DaniMori committed:
Update R version and package dependencies

DaniMori committed:
Update "lattice" package in environment

DaniMori committed:
Merge branch 'development' into draft/sips-2025-version

DaniMori committed:
Drops the "Explore local repo" slide

There is no time to explain everything (only the "renv.lock" file in the next slide).

DaniMori committed:
Explain better the "renv.lock" file

DaniMori committed:
Update software components

DaniMori committed:
Shorten introductory part to save explaining time

DaniMori committed:
Create new section with git's best practices

Moves slide "Do's and dont's to that new section

DaniMori committed:
Reorder sections

Making changes goes first, then configuring git, then the complete git workflow, and then practice with more changes.

DaniMori committed:
Add image for new "git best value" section

DaniMori committed:
Move "versioning in git" section header

DaniMori committed:
Improve codification of notes

Bullet points for parts to read, the rest for clarifications

DaniMori committed:
Update links to materials, hosted in OSF now too

DaniMori committed:
Clarify notes better

DaniMori committed:
Make instruction "non-readable"

It is intended for the instructor only.

DaniMori committed:
Add "save changes" instruction to "make changes"

Some people don't know they have to do this.

DaniMori committed:
Improve analysis output

Avoid overlapping of the tables in the output

JesusSierralaya reviewed (approved):
I agree with all changes

Cristrinaranjus reviewed (approved):
Ready to go!
